cloudify/utils.py

```diff
@@ -543,9 +543,14 @@ def _get_formatted_version(version):
     @staticmethod
     def _get_package_version(plugins_dir, package_name):
         # get all plugin dirs
-        subdirs = next(os.walk(plugins_dir))[1]
+        try:
+            subdirs = next(os.walk(plugins_dir))[1]
+        except StopIteration:
+            return
         # filter by package name
         package_dirs = [dir for dir in subdirs if dir.startswith(package_name)]
+        if not package_dirs:
+            return
```

Comment:
plugins are now not required to be there, in case they're not installed yet, so cope with that